libs/cli/langgraph_cli/cli.py

```diff
@@ -509,6 +521,7 @@ def prepare(
         port=port,
         watch=watch,
         debugger_port=debugger_port,
+        debugger_host=debugger_host or f"http://127.0.0.1:{DEFAULT_PORT}",
```

Comment:
hm, why is the default host not just `127.0.0.1`?

Reply:
(read more, i think it's a bit confusing that we have both host and port, but we're configuring whether the debugger is turned on or not based on port being provided. not sure if this is standard practice, but that would be a bit confusing to me as a user